mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx

```diff
@@ -97,6 +102,33 @@ export function DexScreen (): JSX.Element {
 
   return (
     <>
+      <ThemedView
+        dark={tailwind('bg-gray-800 border-b border-gray-700')}
+        light={tailwind('bg-white border-b border-gray-200')}
+        style={tailwind('flex flex-row px-4 py-3')}
+      >
+        <View style={tailwind('flex flex-col')}>
+          <ThemedText light={tailwind('text-gray-500')} dark={tailwind('text-gray-500')} style={tailwind('text-xs')}>{translate('screens/DexScreen', 'Total Value Locked (USD)')}</ThemedText>
```

Comment:
same label color for both mode 👀?

Reply:
Yes it's based on Section title

Reply:
ah damn it's only for default, others used -400

Reply:
Let me update